cx/slack/README.md

```diff
@@ -58,6 +58,10 @@ You can view a list of your active integration deployments under [Cloud Run](htt
 **Testing the Integration**
 
 *   You have now completed all the necessary steps and can test your integration.
+*   To test rich messages you have to first input in the cutom payload for your agent:
+    *   { "blocks": [ {...} ] } for [images](https://api.slack.com/reference/block-kit/block-elements#image)
+    *   {"text": "...", "attachments": [ {...} ] } for attachments such as [buttons](https://api.slack.com/legacy/message-buttons)
```

Comment:
The document says it's outmoded approach.
> These docs describe an outmoded approach to composing messages and interactive experiences. Learn how to more effectively communicate and help folks get work done by our new documentation and transitioning to "blocks".

Could we use the `blocks` for the button example as the warning indicates?

Reply:
I have restructured the unit tests for button responses and edited the README to make sure we're using the up to date format.